src/fileio_interface.jl

```diff
@@ -1,7 +1,9 @@
 #FileIO Interface
+import FileIO
 
-fileio_save(f, doc::AbstractDict) = bson(f.filename, doc)
-fileio_save(f, args::Pair...) = bson(f.filename, Dict(args))
+fileio_save(f::FileIO.File, doc::AbstractDict) = bson(FileIO.filename(f), doc)
+fileio_save(s::FileIO.Stream, doc::AbstractDict) = bson(FileIO.stream(s), doc)
```

Comment:
Should the signature be tightened to `Stream{format"BSON"}`, according to [FileIO readme](https://github.com/JuliaIO/FileIO.jl)?